src/Microsoft.AspNetCore.OData/Extensions/ODataApplicationBuilderExtensions.cs

```diff
@@ -12,8 +13,20 @@ namespace Microsoft.AspNet.OData.Extensions
     /// </summary>
     public static class ODataApplicationBuilderExtensions
     {
+        private static readonly int MaxTop = 100;
```

Comment:
Is there a variable that is shared with Microsoft.AspNet side? Should it be in the Shared project?

Reply:
It looks no. Thx